lib/index.js

```diff
@@ -11,12 +12,23 @@ function drawCanvas(){
 }
 
 let character = new Character();
+let game = new Game(character);
 
 function drawSprite(){
   context.fillStyle = "green";
   context.fillRect(character.x, character.y, character.width, character.height);
 }
 
+function drawPortal(){
+  context.fillStyle = "papayawhip";
+  context.fillRect(500, 0, 100, 100);
+}
+
+function checkPortal(){
+  //conditional for game state: if game.state===2, make new game-- render new canvas,
```

Comment:
We can give each new game an indexed number to match the state that gets called when our character enters/collides with each portal. We can add as many new portals/games as Pahlka would like to imagine/create.